modules/nw-disable-http2.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-disable-http2_{context}"]
+= Disable HTTP/2
+
+You can disable HTTP/2 on a specific Ingress Controller, or you can disable HTTP/2 for the entire cluster.
+
+.Procedure
+
+* To disable HTTP/2 on a specific Ingress Controller, enter the `oc annotate` command:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator annotate ingresscontrollers/<ingresscontroller_name> ingress.operator.openshift.io/default-enable-http2=false 
+----
++
+Replace `<ingresscontroller_name>` with the name of an Ingress Controller to disable HTTP/2.
+
+* To disable HTTP/2 for the entire cluster, enter the `oc annotate` command:
+*
```

Comment:
```suggestion
+
```

Reply:
I think this is meant to be a `+`, since that will bring the terminal code inline and the asterisks isn't needed.